File: FlashTuna.Core/Modules/Usage/StatisticReportModel.cs
using FlashTuna.Core.Storage;
using System;
using System.Collections.Generic;
using System.Text;

namespace FlashTuna.Core.Modules.Usage
{
    public class StatisticReportModel
    {
        public int DiscoveredMethods { get; set; }
        public int ProblemsCount { get; set; }
        public int ErrorsCount { get; set; }
        public List<TrackableMethodViewModel> Methods { get; set; }
        public string UpdateInterval { get; set; }
        public string TrackingPeriod { get; set; }
    }
}

File: FlashTunaPerfomanceAnalyzer/ClientApp/src/app/models/statistic-report.model.ts
import { TrackableMethodModel } from './trackable-method.model';

export class StatisticReportModel
{
    constructor(discoveredMethods: number,
    errorsCount: number,
    problemsCount: number)
    {
      this.discoveredMethods = discoveredMethods;
      this.errorsCount = errorsCount;
      this.problemsCount = problemsCount;
    };

    discoveredMethods: number;
    errorsCount: number;
  problemsCount: number;
  methods: TrackableMethodModel[];
  trackingPeriod: string;
  UpdateInterval: string;
}

